client/post-editor/editor-gutenberg-opt-in-dialog/style.scss

```diff
@@ -0,0 +1,26 @@
+.editor-gutenberg-opt-in-dialog {
+	overflow-y: hidden;
+}
+
+.editor-gutenberg-opt-in-dialog__left {
+	width: 34%;
+	float: left;
```

Comment:
Let's try to use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) unless this isn't possible. We may not need to do so if we can find an appropriate Calypso container component. @Copons should have a number of tips as well.

Reply:
Updated the styles to use flexbox, the CSS could still use some polish and 👀 from someone who knows how all of this works (will add notes to the PR description).